Podfile

```diff
@@ -5,10 +5,10 @@ target 'iina' do
   use_frameworks!
 
   # Pods for iina
-  pod 'Just', :git => 'https://github.com/JustHTTP/Just.git'
-  pod 'PromiseKit'
+  pod 'Just', :git => 'https://github.com/iina/Just', :branch => 'swift-5'
+  pod 'PromiseKit', :git => 'https://github.com/iina/PromiseKit', :branch => 'xcode-10.2-beta1'
```

Comment:
Upstream added Swift 5 support and seems to work for me.

Reply:
Thanks for letting me know they've added this! I'll update this to point back at the upstream repository.